pepper-apis/src/main/java/org/broadinstitute/ddp/route/UserRegistrationRoute.java

```diff
@@ -237,19 +235,20 @@ private User processInvitation(Response response, Handle handle, StudyDto study,
 
             return user;
         } else if (invitation.getInvitationType() == InvitationType.RECRUITMENT) {
-            var user = signUpNewOperator(response, handle, study, auth0UserId, payload, clientConfig, auth0Util, mgmtClient);
-            invitationDao.assignAcceptingUser(invitation.getInvitationId(), user.getId(), Instant.now());
-            LOG.info("Assigned invitation {} of type {} to user {}", invitation.getInvitationGuid(),
-                    invitation.getInvitationType(), user.getGuid());
-            return user;
+            var pair = signUpNewOperator(response, handle, study, auth0UserId, payload, clientConfig, auth0Util, mgmtClient);
+            invitationDao.assignAcceptingUser(invitation.getInvitationId(), pair.getParticipantUser().getId(), Instant.now());
+            LOG.info("Assigned invitation {} of type {} to participant user {}", invitation.getInvitationGuid(),
+                    invitation.getInvitationType(), pair.getParticipantUser().getGuid());
+            triggerUserRegisteredEvents(handle, study, pair.getOperatorUser(), pair.getParticipantUser());
```

Comment:
I moved triggering events to be outside of `signUpNewOperator()` so that we can do some other bookkeeping (like assign invitation to user) before running events.